.travis.yml

```diff
@@ -238,11 +238,25 @@ _helm_deploy: &helm_deploy
 
     pods_ready
 
+    # Delay running the tests to give ingress & SOLR a chance to fully initialise
+    echo "Waiting 2 minutes from $(date) before running tests..."
+    sleep 120
+
     # run acs checks
     docker run -a STDOUT --volume $PWD/test/postman/helm:/etc/newman --network host postman/newman_alpine33:3.9.2 run "acs-test-helm-collection.json" --global-var "protocol=https" --global-var "url=$namespace.dev.alfresco.me"
+    ACS_TEST_RESULT=$?
+    if [[ "${ACS_TEST_RESULT}" == "1" ]]; then
+      ACS tests failed, exiting
+      exit 1
+    fi
     # run sync service checks
     if [[ "$values_file" != "helm/alfresco-content-services/6.0.N_values.yaml" ]] && [[ "$values_file" != "helm/alfresco-content-services/community_values.yaml" ]]; then
       docker run -a STDOUT --volume $PWD/test/postman/helm:/etc/newman --network host postman/newman_alpine33:3.9.2 run "sync-service-test-helm-collection.json" --global-var "protocol=https" --global-var "url=$namespace.dev.alfresco.me"
+      SYNC_TEST_RESULT=$?
+      if [[ "${SYNC_TEST_RESULT}" == "1" ]]; then
+        Sync tests failed, exiting
+        exit 1
```

Comment:
Dont exit with 1 here @gavincornwell ill will kill the session again, how will the cleanup run then?

Reply:
https://docs.travis-ci.com/user/job-lifecycle you need to use -> TRAVIS_TEST_RESULT

Reply:
I did read about exit 1 not being a good idea after implementing this but we use it in multiple places in the script already :-( 

Also, I tried to use TRAVIS_TEST_RESULT and it doesn't work as advertised, it was always empty, never 0 or 1.  However, even if did work what do we do with the result if we can't exit!?

I'm beginning to hate Travis ;-)